### ISSUE TITLE
Display the seeds used for each query to the 'My queries' page

### ISSUE THREAD

**Issue by p013570** [gchq/gaffer-tools#789]
Perhaps this is hidden by default and a user could click somewhere to see the list of seeds used.

**Comment by n3101** [comment]
The UI fails to meet our Accessibility policy. It is unmaintainable and will be removed shortly.
